src/frontend/src/pages/HomePage.jsx

```diff
@@ -1,8 +1,11 @@
 import { Container, Col, Row } from 'react-bootstrap';
 import { BackgroundContainer, SearchForm } from '@/components';
 import { ToastNotification } from '@/components';
```

Comment:
ToastNotification should live together with like:  import { BackgroundContainer, SearchForm , ToastNotification} from '@/components';